### PR TITLE
doc: update install section

### PR DESCRIPTION
**Pull request by junghans**

### CODE REVIEW

File: guide/README.rst [hunk 1]
```diff
@@ -12,40 +12,20 @@ Installation
...
-
-To build a gromacs version for VOTCA use
-
-.. code:: bash
-
-    cmake -DBUILD_OWN_GROMACS=ON ..
-    make -j5
-    make install
+Instructions how to build and install VOTCA can be found
+`here <https://www.votca.org/INSTALL.html>`__
```
Comment (junghans) [inline guide/README.rst:16]: @JensWehner @felipeZ how would I use `:ref:` here instead of direct link?

Comment (junghans) [reply guide/README.rst:16]: I guess we need labels: https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref

Comment (JensWehner) [reply guide/README.rst:16]: Ahh it should stay inside the manual.

**Review by JensWehner — APPROVED** [review]
Yeah reference is better, but this is already an improvement and so warrants merging.

File: guide/README.rst (hunk 1 above)
Comment (felipeZ) [reply guide/README.rst:16]: You can also use references like the following:
```rst
Then  install RDKit_ and H5PY_:
  - `conda install -y -q -c conda-forge h5py rdkit`


.. _RDKit: https://www.rdkit.org
.. _H5PY: https://www.h5py.org/
```

The underscore at the end of the keyword instructs rst to generate a link to the corresponding link. The last two lines are just shown, just the references.

Comment (junghans) [reply guide/README.rst:16]: @marvinbernhardt do know how to do this?

Comment (marvinbernhardt) [reply guide/README.rst:16]: Adding 
```
+.. _install:
+
 Installation
 ============
```
at the top of votca/share/doc/INSTALL.rst and changing the link here to
```
 Instructions how to build and install VOTCA can be found
-`here <https://www.votca.org/INSTALL.html>`__
+:ref:`here<install>`.
```
works for me. Can I not commit to this branch or do I have my local settings mixed up?

**Comment by junghans** [comment]
@marvinbernhardt github says you have write permission.

File: guide/README.rst [hunk 2]
```diff
@@ -12,40 +12,20 @@ Installation
...
 
 All the tutorials are in the installation folder, i.e.
 ``${prefix}/share/votca/csg-tutorials``.
 
-Downloading the manual
-^^^^^^^^^^^^^^^^^^^^^^
+The manual
+^^^^^^^^^^
+
+The CSG manual can be found `here <https://www.votca.org/csg>`__
```
Comment (junghans) [inline guide/README.rst:27]: @marvinbernhardt can be do the same here?

Comment (marvinbernhardt) [reply guide/README.rst:27]: sure. Btw, I can not create branches on votca/votca

Comment (junghans) [reply guide/README.rst:27]: > sure. Btw, I can not create branches on votca/votca

fixed.

**Comment by marvinbernhardt** [comment]
https://github.com/votca/votca/pull/674
https://github.com/votca/csg/pull/688

**Pull request merged**